source/docs/software/getting-started/options-for-programming.rst

```diff
@@ -68,39 +72,40 @@ Advantages
 
 - Recommended if you're learning or have learned some programming, even better if you know a little Java.
 - Much greater flexibility than Blocks.
-- Much easier to integrate libraries like `EasyOpenCV`_, `FTC Dashboard`_, `FTCLib`_, and `Road Runner`_.
-- Can use plugins like `Road Runner`_.
+- Full suite of modern IDE features (autocomplete, error highlighting, etc) that make working with a text-based programming language convenient
+- Can use libraries like `Road Runner`_, `FTCLib`_, and `FTC Dashboard`_.
 - Can use either a USB connection to the :term:`RC <Robot Controller>` phone, or a wireless connection to upload code.
 
-  .. note:: Deploy times can be sped up by using `OpenRC Turbo`_.
+  .. note:: Deploy times can be sped up significantly by using `Sloth`_.
```

Comment:
I don’t feel comfortable recommending this with the current quality of their documentation; it does not seem approachable. For example, in the README, most of it is jargon that doesn’t mean anything to anyone who has not been actively paying attention to this space. [ i.e., what is sinister and fastload? ]